content/actions/testing.md

```diff
@@ -306,3 +306,59 @@ end
 <p class="notice">
 Please avoid <em>test doubles</em> when writing full integration tests, as we want to verify that the whole stack is behaving as expected.
 </p>
+
+## Session Testing
+
+If action works with session and you need to test this action you can put session to `params` block. For example you have a action with current account:
```

Comment:
I think it'd be better put this way:

"If you need to test an action that works with a session, you can place the session in a `params` block. For example, say you have an action with a `current_account`:"